vcs/Colormap_Create/Color_map_create_own.html

```diff
@@ -1,8 +1,13 @@
 <!DOCTYPE html>
-<html>
-<head><meta charset="utf-8" />
-<title>Color_map_create_own</title><script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.1.10/require.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
+<!-- saved from url=(0056)file:///home/nadeau1/Downloads/Color_map_create_own.html -->
+<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+
+<title>Color_map_create_own</title>
+
+<script src="./Color_map_create_own_files/require.min.js"></script>
```

Comment:
Not sure it is a good idea to change away from the direct links for the require.min.js and jquery.min.js files - is this an artifact of having the html file saved from the local copy in the Downloads folder?